modules/ROOT/pages/update-acm.adoc

```diff
@@ -10,7 +10,7 @@ Upgrades with new features are applied to sandbox environments four weeks before
 
 To ensure that the community administrator controls how their audience receives community changes, new functionality in upgrades is typically disabled until the community administrator enables it.
 
-The end-of-life period for package version 1.x is from August 12, 2021 to August 12, 2022. During this period, no new 1.x versions or patches are released, and only end-of-life support is available. All support for version 1.x ends August 12, 2022.
+API Community Manager follows the standard support policy for MuleSoft cloud offerings, and only the latest version of API Community Manager 2.x is supported. API Community Manager 1.x reached End of Life (EOL) on August 14, 2021 and is not supported.
```

Comment:
Is the standard support policy = supporting only the latest version? Or is the first part of the sentence actually different from the second (following ", and")?

Reply:
Also...does this mean that if the latest version of ACM is 2.5, neither 2.4 and earlier, and of course 1.x versions, are supported?

Reply:
If ^ are true, can we use
```suggestion
Because API Community Manager follows the standard support policy for MuleSoft cloud offerings, only the latest version of API Community Manager 2.x is supported. Additionally, API Community Manager 1.x reached End of Life (EOL) on August 14, 2021.
```

Reply:
Correct, the standard policy is that the latest version (2.5 in the example) is supported, 2.4 and earlier aren't, and 1.x aren't. That edit looks great, and I've updated the PR to use it.